.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: elixir
+
+install:
+  - mix local.rebar --force
+  - mix local.hex --force
+  - mix deps.get
+  - mix compile
```

Comment:
Do we need line 7 if we have line 8? Perhaps there's a difference I'm not aware of (couldn't find anything in documentation).